Blockzilla/SearchEngine.swift

```diff
@@ -35,8 +35,9 @@ class SearchEngine: NSObject, NSCoding {
     }
 
     func urlForSuggestions(_ query: String) -> URL? {
+        let trimmed = query.trimmingCharacters(in: .whitespaces)
         if let suggestTemplate = suggestionsTemplate {
-            if let escaped = query.addingPercentEncoding(withAllowedCharacters: .urlQueryParameterAllowed) {
```

Comment:
I know you didn't set up this structure, but I think having one joined guard statement would make this much cleaner. (ie. 
`guard let suggestTemplate = suggestionsTemplate, let escaped = trimmed.addingPercentEncoding(withAllowedCharacters: .urlQueryParameterAllowed) else { return nil }`

The structure of having the entire function wrapped in two if lets is not ideal.

Reply:
Agreed, was looking to minimize lines changed, but this should definitely be fixed (especially since the other "urlFor" function uses guard statements)